Aardvark/ARKEmailBugReporter.h

```diff
@@ -53,6 +54,20 @@
 @end
 
 
+typedef void (^ARKEmailBugReporterCustomPromptCompletionBlock)(ARKEmailBugReportConfiguration *_Nullable configuration);
+
+@protocol ARKEmailBugReporterPromptingDelegate <NSObject>
+
+@required
+
+/// Called on the main thread when a bug is filed to signal that a bug report prompt should be presented to the user.
+/// The `completion` should be called with either an updated configuration to present the email dialog, or `nil` to signal that the prompt was cancelled.
+/// When the initial `configuration` has `includesScreenshot` or `includesViewHierarchyDescription` false, setting the field to true will have no effect.
```

Comment:
"_set to_ false"?

But also: I'm not sure I get this comment. Why does changing some fields work, and some other fields not? Where should `includesScreenshot` or `includesViewHierarchyDescription` be set? Kinda feels like `includesScreenshot` and `includesViewHierarchyDescription` shouldn't be mutable... but maybe I'm missing something.

Reply:
Basically it's a one-way door.  If there is a screenshot, the initial configuration will have `includesScreenshot` set to `true`.  The prompt can leave this alone to include the screenshot or set it to `false` so the screenshot won't be included.  If there is no screenshot to include, the initial configuration will have `includesScreenshot` set to `false`.  Setting it to `true` will have no effect since there's nothing to attach.

Reply:
So then, why are these fields mutable at all?

Reply:
We still want the prompt to be able to remove the screenshot and view hierarchy.  If we want to make the API more explicit here, we could do something like:
```objc
/// Whether or not there is a screenshot attached to the bug report.
@property (nonatomic, readonly) BOOL includesScreenshot;

/// Removes the screenshot from the bug report if one is attached.
- (void)removeScreenshot;

/// Whether or not there is a view hierarchy description attached to the bug report.
@property (nonatomic, readonly) BOOL includesViewHierarchyDescription;

/// Removes the view hierarchy description from the bug report if one is attached.
- (void)removeViewHierarchyDescription;
```

Alternatively, we could just attach the screenshot data, which would have the added advantage of allowing the custom prompt to modify the image.  Although this would probably be dependent on #67 (and maybe #66).
```objc
@property (nullable, nonatomic, copy) UIImage *screenshot;

@property (nullable, nonatomic, copy) NSString *viewHierarchyDescription;
```

Reply:
Talked about this offline with @efirestone and landed on going to `remove*` route for now, and eventually moving to including the screenshot and hierarchy themselves with some other breaking changes in the next major release.